src/System.IO/tests/StreamWriter/StreamWriter.CtorTests.cs

```diff
@@ -31,15 +31,6 @@ public static void CreateStreamWriter()
             Assert.Equal("HelloWorld", str2);
         }
 
-        [Fact]
-        public static void NullEncodingThrows()
-        {
-            // [] Check for ArgumentNullException on null encoding
-            //-----------------------------------------------------------------
-
-            Assert.Throws<ArgumentNullException>(() => new StreamWriter(new MemoryStream(), null));
-        }
```

Comment:
This should now have the opposite test, validating that null can be passed in. And there should be similar tests for the other optional arguments.

Reply:
Please suggest how I can test this. Should I just use the constructor like this:
```csharp
[Fact]
public void StreamReader_WithOptionalArguments()
{
    using (var sr = new StreamReader(new MemoryStream(), leaveOpen: true)) { }
    using (var sr = new StreamReader(new MemoryStream(), encoding: null)) { }
    using (var sr = new StreamReader(new MemoryStream(), bufferSize: -1)) { }
    using (var sr = new StreamReader(new MemoryStream(), detectEncodingFromByteOrderMarks: true)) { }
}
```

Reply:
You should keep this test and mark it so it only runs on the desktop .NET Framework (e.g. 4.7.2 etc.).  We want to still test the old behavior there. You do this by adding the following attribute:

``` C#
[SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)] // e.g. NetFX only
```